cache_images/fedora_setup.sh

```diff
@@ -17,6 +17,12 @@ fi
 # shellcheck source=./lib.sh
 source "$REPO_DIRPATH/lib.sh"
 
+# Make /tmp tmpfs bigger, by default we only get 50%. Bump it to 75% so the tests have more storage.
+# Do not use 100% so we do not run out of memory for the process itself if tests start leaking big
+# files on /tmp.
+$SUDO mkdir /etc/systemd/system/tmp.mount.d
```

Comment:
Suggestion: `mkdir -p`, so command won't fail if (some day) that directory already exists